src/patternfly/components/TextInputGroup/text-input-group.scss

```
@@ -22,30 +22,31 @@
   --#{$text-input-group}__text--BorderEndStartRadius: var(--#{$text-input-group}__text--BorderRadius--base);
 
   // Label group
-  --#{$text-input-group}--c-label-group__main--PaddingBlockStart: var(--pf-t--global--spacer--xs);
-  --#{$text-input-group}--c-label-group__main--PaddingInlineEnd: var(--pf-t--global--spacer--xs);
-  --#{$text-input-group}--c-label-group__main--PaddingBlockEnd: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}--c-label-group__main--PaddingBlockStart: calc(var(--pf-t--global--spacer--control--vertical--default) / 2);
+  --#{$text-input-group}--c-label-group__main--PaddingInlineEnd: calc(var(--pf-t--global--spacer--control--horizontal--plain) / 2);
+  --#{$text-input-group}--c-label-group__main--PaddingBlockEnd: calc(var(--pf-t--global--spacer--control--vertical--default) / 2);
 
   // Text input
-  --#{$text-input-group}__text-input--PaddingBlockStart: var(--pf-t--global--spacer--sm);
-  --#{$text-input-group}__text-input--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
-  --#{$text-input-group}__text-input--PaddingBlockEnd: var(--pf-t--global--spacer--sm);
-  --#{$text-input-group}__text-input--PaddingInlineStart: var(--pf-t--global--spacer--sm);
+  --#{$text-input-group}__text-input--PaddingBlockStart: var(--pf-t--global--spacer--control--vertical--default);
+  --#{$text-input-group}__text-input--PaddingInlineEnd: var(--pf-t--global--spacer--control--horizontal--plain);
```

I think this should just use the default var?

```suggestion
  --#{$text-input-group}__text-input--PaddingInlineEnd: var(--pf-t--global--spacer--control--horizontal--default);
```